ringtone_gwynn.R

```diff
@@ -0,0 +1,24 @@
+# Create your own ringtone! 
+
+library(Rmusic) # Install it with remotes::install_github("keithmcnulty/Rmusic")
+
+seed <- 62   #choose a seed
+n_notes <- 5 #choose a integer between 1 and 5
```

Comment:
```suggestion
n_notes <- 6 #choose a integer between 1 and 5
```
What do you think about using 6 notes